File: core/src/com/t2wonderland/kurona/Models/WorldListener.java
package com.t2wonderland.kurona.models;

import com.t2wonderland.kurona.Assets;
import com.t2wonderland.kurona.interfaces.IWorldListener;

public class WorldListener implements IWorldListener {
    @Override
    public void sp() {
        // 技発動サウンド
        Assets.playSound(Assets.spSound);
    }

    @Override
    public void food() {
        // 食べ物たべるサウンド
        Assets.playSound(Assets.foodSound);
    }
}

File: core/src/com/t2wonderland/kurona/Models/WorldListener.kt
package com.t2wonderland.kurona.Models

import com.t2wonderland.kurona.Assets
import com.t2wonderland.kurona.Interfaces.IWorldListener

class WorldListener : IWorldListener {
    override fun sp() {
        Assets.playSound(Assets.spSound)
    }

    override fun food() {
        Assets.playSound(Assets.foodSound)
    }
}
